lib/metrics.js

```diff
@@ -18,6 +18,12 @@ const fs = require('fs');
 
 const DEFAULT_METRIC_TIMEOUT_MS = 60000; // default openwhisk action timeout
 const DEFAULT_MAX_STRING_LENGTH = 100;
+const DEFAULT_ERROR_METRIC_MAX_STRING_LENGTH = 1500;
```

Comment:
Optional suggestion: What about adding an environment variable for this? Having longer messages could be useful for debugging... Wdyt?